pages/_app.tsx

```diff
@@ -61,14 +70,29 @@ const App = ({ Component, pageProps }) => {
     AOS.init({ duration: 1200 });
   }, []);
 
+  const [queryClient] = useState(
+    () =>
+      new QueryClient({
+        defaultOptions: { queries: { staleTime: FIVE_MINS_MS } },
+      })
+  );
+
   return (
     <>
       <Analytics />
       <DefaultSeo {...NEXT_SEO_DEFAULT} />
       <AppInsightsProvider>
-        <ErrorBoundary key={router.asPath}>
-          <Component {...pageProps} />
-        </ErrorBoundary>
+        <QueryClientProvider client={queryClient}>
+          <ErrorBoundary key={router.asPath}>
+            <Component {...pageProps} />
+          </ErrorBoundary>
+          {process.env.NODE_ENV === "development" && (
+            <ReactQueryDevtoolsProduction
```

Comment:
ReactQueryDevtoolsProduction is not an actual production, maybe change this to something else to reflect its environment